internal/cli/main.go

```diff
@@ -54,7 +54,7 @@ Commands:
   goto V       Migrate to version V
   up [N]       Apply all or N up migrations
   down [N]     Apply all or N down migrations
-  drop         Drop everything inside database
+  drop         Drop everything inside database (use -f to bypass confirmation)
```

Comment:
Please follow the existing format. e.g. `drop [-f]`
See `create`, `up`, and `down` for examples